### PR TITLE
[AIFBR-11855] add missing 22.4 oob models

### PR DESCRIPTION
**Pull request by vivekjhauipath**

### CODE REVIEW

**Review by mohitseth-nitrr — DISMISSED** [review]
LGTM

**Review by vnaren23 — APPROVED** [review]
LGTM

File: metadata/CustomNamedEntityRecognition__8__metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "inputType": "JSON",
+    "displayName": "CustomNamedEntityRecognition",
+    "memory": 0,
+    "mlPackageLanguage": "PYTHON38",
+    "projectId": "[project-id]",
+    "stagingUri": "[staging-uri]",
+    "projectName": "UiPath Language Analysis",
+    "projectDescription": "Curated Models from UiPath to analyze language in Emails, Documents, Web Pages, Text Messages, Portals and more",
+    "tenantName": "UiPath",
+    "minAIFabricVersion": "v22.4",
```
[the remark this answers is not in the document]
Comment (vivekjhauipath) [reply metadata/CustomNamedEntityRecognition__8__metadata.json:20]: sankar added, Anything is fine I guess

**Review by vnaren23 — APPROVED** [review]

**Review by mohitseth-nitrr — APPROVED** [review]
LGTM

**Pull request merged**